packages/ui/src/coreModules/form/components/FieldTemplate/index.js

```diff
@@ -1,16 +1,59 @@
-import React from 'react'
+import { isEmpty } from 'lodash'
```

Comment:
by our unofficial convention we keep React/redux on top, so lodash should come somewhere after the import from redux